Sources/StreamChatUI/Composer/ChatMenu/EmojiMenu/GifViewController.swift

```diff
@@ -0,0 +1,383 @@
+//
+//  EmojiMainViewController.swift
+//  StreamChatUI
+//
+//  Created by Mohammed Hanif on 26/04/22.
+//
+
+import UIKit
+import GiphyUISDK
+import Combine
+
+@available(iOS 13.0, *)
+class GifViewController: UIViewController {
+
+    open private(set) lazy var searchView = UISearchBar
+        .init()
+        .withoutAutoresizingMaskConstraints
+
+    open private(set) lazy var backButton = UIButton
+        .init()
+        .withoutAutoresizingMaskConstraints
+
+    open private(set) lazy var hStack = UIStackView
```

Comment:
@mdhanif-simformsolutions  variable name should be in descriptive way.

Reply:
Updated this. thank you.